src/PolygonDraw/validators.ts

```diff
@@ -1,8 +1,4 @@
 import { Coordinate } from 'types';
 import { isPolygonClosed } from '../helpers';
 
-export const isValidPolygon = (polygon: Coordinate[]) => (
-    polygon &&
-    polygon.length > 3 &&
-    isPolygonClosed(polygon)
-);
+export const isValidPolygon = (polygon: Coordinate[]) => polygon && polygon.length > 3 && isPolygonClosed(polygon);
```

Comment:
Question: do we need to check 'polygon && ...' , I mean polygon is an array as base type and that mean it's always defined , and also as an argument it could not be defined , so I think it's not needed in this case.

Reply:
Since its an exposed function, it will also be used outside of the library. That means, it can be used in JS also, which can result into use it with null pretty easy. Even in TS its still not safe to not be called with undefined in runtime, since you could use a mapped API response object directly where the reference used is unset. Both will result into an unhandled error, which would be a bad case of a check method like that. I would therefore avoid removing it and keep the current pattern since it is also a really easy catch on that problem.